client/modules/User/pages/DashboardView.jsx

```diff
@@ -126,63 +100,60 @@ class DashboardView extends React.Component {
           <SketchList key={username} mobile={mobile} username={username} />
         );
     }
-  }
-
-  render() {
-    const currentTab = this.selectedTabKey();
-    const isOwner = this.isOwner();
-    const { username } = this.props.params;
-    const actions = this.renderActionButton(currentTab, username, this.props.t);
-
-    return (
-      <RootPage fixedHeight="100%">
-        <Nav layout="dashboard" />
-
-        <main className="dashboard-header">
-          <div className="dashboard-header__header">
-            <h2 className="dashboard-header__header__title">
-              {this.ownerName()}
-            </h2>
-            <div className="dashboard-header__nav">
-              <DashboardTabSwitcherPublic
-                currentTab={currentTab}
-                isOwner={isOwner}
-                username={username}
-              />
-              {actions && (
-                <div className="dashboard-header__actions">{actions}</div>
-              )}
-            </div>
-          </div>
+  };
+
+  const currentTab = selectedTabKey();
+  const owner = isOwner();
+  const { username } = params;
+  const actions = renderActionButton(currentTab, username, t);
```

Comment:
Since you eliminated the unnecessary arguments from the function, you no longer need to pass them here.